### PR TITLE
Add FAQ entry about requesting new Trove classifiers

### LINKED ISSUE #3028: Add "how to request new trove classifier" item to FAQ
[This tweet](https://twitter.com/pydanny/status/966454907699187712) reminded me: http://pypi.org/help/ should include an item telling people to file a GitHub issue here to request a new trove classification.

### PR DESCRIPTION
**Pull request by jonparrott**
Fixes #3028

### CODE REVIEW

**Comment by nlhkabu** [comment]
ping @brainwane on this.  Maybe we should also add a new github issue template for this?

**Comment by jonparrott** [comment]
> Can we be more specific about what we need from the requester?

Sure - beyond the name of the classifier is there any other info we need?

**Comment by dstufft** [comment]
We generally ask for some justification on why a classifier should be added. We have an already long list and we don’t want to inflate them further with (more) classifiers that nobody would use.

**Comment by jonparrott** [comment]
More details added in 39a99e8e

**Comment by brainwane** [comment]
>  Maybe we should also add a new github issue template for this?

This sounds like a good idea! "Adding a trove classifier? (a) name of new classifier (b) why don't the existing classifiers fit this need?"

File: warehouse/templates/pages/help.html [hunk 1]
```diff
@@ -246,6 +248,11 @@ <h3 id="file-name-reuse">{{ file_name_reuse() }}</h3>
           To avoid this situation, <a href="https://packaging.python.org/guides/using-testpypi/">use Test PyPI to perform and check your upload first</a>, before uploading to <a href="https://pypi.org">pypi.org</a>.
         </p>
 
+        <h3 id="new-classifier">{{ new_classifier() }}</h3>
+        <p>
+          If you would like to request a new Trove classifer please file a bug on our <a href="https://github.com/pypa/warehouse/issues" target="_blank" rel="noopener">issue tracker</a>. Please include the name of the requested classifier and a brief justification of why it is important.
+        </p>
+
```
Comment (brainwane) [inline warehouse/templates/pages/help.html]: Please add the `{% macro code_of_conduct() %}` block here as it is in other similar answers that are primarily about filing an issue. Thanks.

Comment (jonparrott) [reply warehouse/templates/pages/help.html]: Done in d230e291

**Comment by jonparrott** [comment]
Okay, I think this is about ready to merge.

Screenshot:

screen shot 2018-03-27 at 2 04 54 pm

**Comment by nlhkabu** [comment]
Thanks @jonparrott !

Following from my previous comment - could you please add a new GitHub template (see `https://github.com/pypa/warehouse/blob/master/.github/ISSUE_TEMPLATE/visual-design.md` as an example).

It should be named 'request-trove-classifier.md' and include something like:

```md
## The name of the classifier you would like to add
<-- Classifier name -->

## Why do you want to add this classifier?
<!-- Please include a brief explanation to justify your request. Why do the current classifiers not meet your need? -->
```

**Then** we can change th link to the issue tracker to use this GitHub template.  It would also be cool if we could  apply the 'classifier request' badge. The docs for how to do this are here: https://help.github.com/articles/about-automation-for-issues-and-pull-requests-with-query-parameters/

Let me know if anything is unclear, or you need help :)

**Comment by jonparrott** [comment]
@nlhkabu great idea. Done in 8ee6c860

> It would also be cool if we could apply the 'classifier request' badge.

Turns out GitHub doesn't allow non-project members to set labels, so I couldn't make this happen.

File: .github/ISSUE_TEMPLATE/new-trove-classifier.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Request to add a new Trove classifier.
+
+## The name of the classifier you would like to add
+<!-- Classifier name -->
+
+## Why do you want to add this classifier?
+<!--
+    Please include a brief explanation to justify your request.
+    Why do the current classifiers not meet your need?
+    How many packages to you expect to use this new classifier?
```
Comment (brainwane) [inline .github/ISSUE_TEMPLATE/new-trove-classifier.md]: Hey, I'm a little unclear -- do you mean how many different PyPI *projects*? I figure usually that's what we want to ask, but I am open to being told differently!

File: warehouse/templates/pages/help.html [hunk 3]
```diff
@@ -246,6 +248,12 @@ <h3 id="file-name-reuse">{{ file_name_reuse() }}</h3>
           To avoid this situation, <a href="https://packaging.python.org/guides/using-testpypi/">use Test PyPI to perform and check your upload first</a>, before uploading to <a href="https://pypi.org">pypi.org</a>.
         </p>
 
+        <h3 id="new-classifier">{{ new_classifier() }}</h3>
+        <p>
+          If you would like to request a new Trove classifier please file a bug on our <a href="https://github.com/pypa/warehouse/issues/new?template=new-trove-classifier.md" target="_blank" rel="noopener">issue tracker</a>. Please include the name of the requested classifier and a brief justification of why it is important.
```
Comment (brainwane) [inline warehouse/templates/pages/help.html]: I like this wording -- thanks.

**Comment by nlhkabu** [comment]
> Turns out GitHub doesn't allow non-project members to set labels, so I couldn't make this happen.

:(  I'll make a separate PR once this is merged.

File: .github/ISSUE_TEMPLATE/new-trove-classifier.md (hunk 2 above)
Comment (jonparrott) [reply .github/ISSUE_TEMPLATE/new-trove-classifier.md]: Nah, project is the right terminology. You would think that I would be better about this. :)

Fixed in ba7560f8

**Pull request merged**